### DISCUSSION TITLE
How does the 'condition' block work?

### DISCUSSION THREAD

**Discussion by Jiuhk** [AutomaApp/automa#175] [Q&A] [answered]
In the 'condition' block I can add a new condition to compare a value to something, e.g. == 1 or >= 10. But, how do I specify which value I'm using to compare to in the first place? Couldn't find documentation on this one.

FYR I would like to compare using a variable defined in a previous 'JavaScript Code' block. Thank you.

**Comment by Kholid060** [comment, marked as answer]
In v0.9.3, you can define the value that you want to compare. And in this case, to get the previous value, you can write `{{ prevBlockData }}` in the first input.

![image](https://user-images.githubusercontent.com/22908993/145737035-eb5c9dbb-da8d-473e-b380-fdba9b904429.png)

**Reply by Jiuhk** [reply]
Great! Thanks man, this is absolutely a cool work to me.
